examples/README.md

```diff
@@ -15,5 +15,52 @@ cd examples/arduino-uno
 cargo run --bin uno-blink
 ```
 
-You need to install [`ravedude`](https://crates.io/crates/ravedude) to make
+You need to install [`ravedude`](https://crates.io/crates/ravedude) with `cargo install ravedude` to make
 this work.
+
+The Examples
+------------
```

Comment:
I think this list is just prone to be outdated all the time... I would rather add a small blurb of information to the top of each example.  That said, I would suggest you just remove this text from the README for now and I'll go back later and sync all the examples.

Reply:
I know what you mean! I assembled the list because I would have liked this kind of list for an overview when learning the basics. I'd also say that it doesn't matter if it is not up to date (that's why I did not add a list of which example is implemented for which board - that list would have been very hard to maintain).

Reply:
> would have liked this kind of list for an overview when learning the basics.

This is valuable feedback, thanks a lot.  I'll consider what we can do here...